### DISCUSSION TITLE
Unable to compile the project

### DISCUSSION THREAD

**Discussion by theoreticalphysics-cpu** [MadAnalysis/madanalysis5#252] [Q&A]
Hi, 

When attempting to download the standalone version of madanalysis5? I am encountering an error that the project is unable to compile. Attached is the error log. 
![Screenshot from 2024-04-28 03-00-54](https://github.com/MadAnalysis/madanalysis5/assets/85380045/b92b338c-fb2b-4d1c-a61b-68b643a0649b)
![Screenshot from 2024-04-28 03-00-48](https://github.com/MadAnalysis/madanalysis5/assets/85380045/7acbcfce-6266-4cf8-85a3-530e31c9f781)
[compilation_root.log](https://github.com/MadAnalysis/madanalysis5/files/15141164/compilation_root.log)

**Comment by BFuks** [comment]
Hi,

It seems that root executables cannot be found by the code. Please install root properly and then restart the code. Everything should then work.

Regards,

Benjamin

**Reply by theoreticalphysics-cpu** [reply]
I have attempted to reinstall root, however I am now encountering the following error. 
[linking_root.log](https://github.com/MadAnalysis/madanalysis5/files/15146834/linking_root.log)

![Screenshot from 2024-04-29 03-43-51](https://github.com/MadAnalysis/madanalysis5/assets/85380045/1ab96e57-a89a-4966-ad76-dc8676b32205)
I am not sure what to do about this. For the record, my root installation seems to be normal since it starts properly when I use the command "root" in the terminal.

**Reply by theoreticalphysics-cpu** [reply]
Hi, 
It seems to have been resolved after I reinstalled ROOT for the fifth time.

Thanks for the assistance.
